drivers/can/can_mcan_int.h

```diff
@@ -1084,7 +1084,7 @@
 
 /* Rx FIFO 0 Fill Level */
 #define CAN_MCAN_RXF0S_F0FL_POS      (0U)
-#define CAN_MCAN_RXF0S_F0FL_MSK      (0x3FUL << CAN_MCAN_RXF0S_F0FL_POS)
```

Comment:
Do you need this change?
It seems not to affect the stm32g4, because the next 4 bits are reserved (0 default value)

Reply:
According to the M_CAN reference manual, the `F0FL` field in the `RXF0S` register is indeed 7 bit wide, not 6. The correct mask is `0x7FUL`.

![image](https://user-images.githubusercontent.com/1076226/148698903-2fc3c133-f03d-47f0-abb2-b5a30caf3949.png)

Reply:
Yes, but it was changed in the stm32fdcan part, wehere it is 6 bit wide

Reply:
This change doesn't impact the stm32fdcan, and matches both the M_CAN reference manual and the stm32h7 reference manual.